src/ResourceManager/ServiceBus/Commands.ServiceBus/help/Get-AzureRmServiceBusDRConfigurations.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: Microsoft.Azure.Commands.ServiceBus.dll-Help.xml
+Module Name: AzureRM.ServiceBus
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmServiceBusDRConfigurations
+
+## SYNOPSIS
+Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## SYNTAX
+
+```
+Get-AzureRmServiceBusDRConfigurations [-ResourceGroupName] <String> [-Namespace] <String>
+ [[-Name] <String>] [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmServiceBusDRConfigurations** Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmServiceBusDRConfigurations -ResourceGroupName "SampleResourceGroup" -Namespace "SampleNamespace_Primary" -Name "SampleDRCongifName"
```

Comment:
@v-Ajnava for examples in your markdown files, please make sure to provide the corresponding output to the cmdlet calls so users will have an idea of what kind of properties that returned object has

Reply:
Resolved.